src/book/03-guides/01-main-codebase/01-getting-started.mdx

```diff
@@ -235,10 +303,197 @@ To run the code on a robot, you need to make sure the robot is powered on, insta
 
    This will start the binary to run the stand script, but the robot won't move until you press the green middle button on its back.
```

Comment:
Should we mention what the red button does too?

Reply:
Probably worth mentioning for safety reasons.